api/backend/blueflood/blueflood.go

```diff
@@ -119,8 +130,11 @@ func (b *blueflood) FetchSingleSeries(request api.FetchSeriesRequest) (api.Times
 	if !ok {
 		return api.Timeseries{}, fmt.Errorf("unsupported SampleMethod %s", request.SampleMethod.String())
 	}
+	queryResolution := b.config.bluefloodResolution(
+		time.Duration(request.Timerange.Resolution()),
```

Comment:
The `.Resolution()` method on `Timerange` objects should already be returning a `time.Duration` type, so the conversion can be omitted.